sst/weatherForecast/Weather_Forecast.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>Remove identation since this block of code. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/reports-performance-summary/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Fixed</p>